[CHAL-11] updated border color and font size of rounded input

ekarthiek opened the pull request:
**The following must be included in every PR. Please check all items before submitting your request.
If an item can not be satisfied, please provide a brief explanation as to why the item is not included.**

- [x] Rebased from Master prior to creation of pull request
- [ ] Minimum 1 unit test that covers the component
- [x] All tests run and passed prior to creation of pull request
- [x] Component added to bit.dev server
- [x] New feature documentation added to bit.dev
- [x] Ran `yarn sync` to get most recent version of all components from bit.dev server
- [x] Sandbox example of your component added/updated to App.js

**Provide a brief description of your update:**
 - JIRA Link:https://campgladiator.atlassian.net/browse/CHAL-11
 - Figma Link:https://www.figma.com/file/vvqFoOfZ5B5AO9VkhpNnth/CGUI-Atoms-Library?node-id=1756%3A836
 - Added two variables to variables.scss file
 - Updated the border color,border width, color, color of placeholder text, font-size
 - The component already has the unit test cases for this requirement in (src/components/atoms/Input/Input.spec.js)

korokd commented:
There were changes only to the `variables` and the `Input`. Why were so many components' version upped?

ekarthiek commented:
> There were changes only to the `variables` and the `Input`. Why were so many components' version upped?

Its a auto-tagged dependency.. Since variable.scss has been refernced in multiple components that’s why the version are getting updated for other components

korokd commented:
> > There were changes only to the `variables` and the `Input`. Why were so many components' version upped?
> 
> Its an auto-tagged dependency. Since variables.scss has reference in multiple components that’s why the version are getting updated for other components

So, if I use the older version of the component, it would use the older version of variables; and with the newer version of the component, the newer version of the variables. Is that it?

korokd reviewed (dismissed):
Just noticed that the `Minimum 1 unit test that covers the component` is checked but there are no new tests in this PR.

If, for some reason, it does not apply to this PR to add tests, then please uncheck it and explain on the PR description. Otherwise, please add at least 1 unit test, as the check requires.

ekarthiek commented:
> > > There were changes only to the `variables` and the `Input`. Why were so many components' version upped?
> > 
> > 
> > Its an auto-tagged dependency. Since variables.scss has reference in multiple components that’s why the version are getting updated for other components
> 
> So, if I use the older version of the component, it would use the older version of variables; and with the newer version of the component, the newer version of the variables. Is that it?

Yes, that's right

ekarthiek commented:
> Just noticed that the `Minimum 1 unit test that covers the component` is checked but there are no new tests in this PR.
> 
> If, for some reason, it does not apply to this PR to add tests, then please uncheck it and explain on the PR description. Otherwise, please add at least 1 unit test, as the check requires.

I unchecked it

korokd commented:
> > Just noticed that the `Minimum 1 unit test that covers the component` is checked but there are no new tests in this PR.
> > If, for some reason, it does not apply to this PR to add tests, then please uncheck it and explain on the PR description. Otherwise, please add at least 1 unit test, as the check requires.
> 
> I unchecked it

Please add an explanation to the PR description as to why there is no new test when the template requests it.

websmith reviewed (changes requested):
This PR doesn't fully support the changes that are needed for the input fields based on what I'm seeing in Figma. Here are some issues I'm still seeing:

- according to Figma, the rounded input field should now be the default input field. There doesn't appear to be an example of an input field without the rounded border anymore. To  support this, we could remove the --rounded modifier and make that the default for the input field.
- The font size and weight still do not match what are in Figma, which shows 12px font with 300/book weight. The code still shows 14px at 400/normal weight.

websmith reviewed (changes requested):
@ekarthiek I also noticed the placeholder text is all-caps, which doesn't match figma

ekarthiek commented:
> This PR doesn't fully support the changes that are needed for the input fields based on what I'm seeing in Figma. Here are some issues I'm still seeing:
> 
> * according to Figma, the rounded input field should now be the default input field. There doesn't appear to be an example of an input field without the rounded border anymore. To  support this, we could remove the --rounded modifier and make that the default for the input field.
> * The font size and weight still do not match what are in Figma, which shows 12px font with 300/book weight. The code still shows 14px at 400/normal weight.

@websmith In this update I am focusing on the default version and in that version the font size is 14px and I have updated the font weight

ekarthiek commented:
> > This PR doesn't fully support the changes that are needed for the input fields based on what I'm seeing in Figma. Here are some issues I'm still seeing:
> > 
> > * according to Figma, the rounded input field should now be the default input field. There doesn't appear to be an example of an input field without the rounded border anymore. To  support this, we could remove the --rounded modifier and make that the default for the input field.
> > * The font size and weight still do not match what are in Figma, which shows 12px font with 300/book weight. The code still shows 14px at 400/normal weight.
> 
> @websmith 
- I have updated the component with the latest design as default and I removed all the modifiers
- In this update I am focusing on the default version and in that version the font size is 14px and I have updated the font weight

ekarthiek commented on src/components/atoms/Input/Input.scss:
```diff
@@ -57,12 +57,26 @@ $input-border: 2px solid $light-gray;
+
+    &:focus {
+      border-color: $navy-lighter;
+      box-shadow: 0 0 0 3px rgba(167,204,237,.5);
```
@websmith I have updated the box-shadow with the $navy-lighter variable

ekarthiek commented on src/components/atoms/Input/Input.scss:
```diff
@@ -57,12 +57,26 @@ $input-border: 2px solid $light-gray;
+    }
+
+    &:focus {
+      border-color: $navy-lighter;
```
> @ekarthiek The border-color itself, according to Figma is using a different color than what you have assigned. The correct color should be #3498DB. Can you add this to the sass color variables as $navy-light to match Figma?

@websmith I changed the border-color to $navy-light

ekarthiek commented on src/assets/styles/variables.scss:
```diff
@@ -6,6 +6,8 @@ $medium-gray: #99999a;
 $lighter-gray: #ededed;
 $lightest-gray: #f9f9f9;
+$text-gray: #656566;
+$navy-lighter:#A7CCED;
```
> @ekarthiek For better readability, this would be a good opportunity to move all similar colors within proximity of eachother?
> 
> **Example:**
> 
> ```
> $gladiator: #c8102e;
> $gladiator-gradient: linear-gradient(215.85deg, #ea3855 9.33%, #c8102e 97.69%);
> 
> $pavement: #231f20;
> $dark-gray: #333;
> $text-gray: #656566;
> $medium-gray: #99999a;
> $light-gray: #d4d4d4;
> $lighter-gray: #ededed;
> $lightest-gray: #f9f9f9;
> 
> $navy: #263746;
> $navy-light: #3498dB;
> $navy-lighter:#A7cced;
> ```

@websmith I moved the variables with similar colors within proximity of each other

ekarthiek commented:
> > > Just noticed that the `Minimum 1 unit test that covers the component` is checked but there are no new tests in this PR.
> > > If, for some reason, it does not apply to this PR to add tests, then please uncheck it and explain on the PR description. Otherwise, please add at least 1 unit test, as the check requires.
> > 
> > 
> > I unchecked it
> 
> Please add an explanation to the PR description as to why there is no new test when the template requests it.



> @ekarthiek I also noticed the placeholder text is all-caps, which doesn't match figma

@websmith Shall I know in which place you are facing this issue. I am not facing such issue

websmith commented:
@ekarthiek Did you remember to push your changes up to remote? I don't see any changes to this branch

websmith commented:
> > > > Just noticed that the `Minimum 1 unit test that covers the component` is checked but there are no new tests in this PR.
> > > > If, for some reason, it does not apply to this PR to add tests, then please uncheck it and explain on the PR description. Otherwise, please add at least 1 unit test, as the check requires.
> > > 
> > > 
> > > I unchecked it
> > 
> > 
> > Please add an explanation to the PR description as to why there is no new test when the template requests it.
> 
> > @ekarthiek I also noticed the placeholder text is all-caps, which doesn't match figma
> 
> @websmith Shall I know in which place you are facing this issue. I am not facing such issue

Nevermind this one. I was looking at the dropdown instead of the input field which is not relevant to this task

PrashantKIsHere reviewed (approved):
Approved
